Pipfile

```diff
@@ -42,15 +42,15 @@ django-sesame = "*"
 dc-api-common = {path = "./api/common"}
 boto3 = "*"
 mangum = "*"
+setuptools = "*"
+django = "==4.2.16"
```

Comment:
I've been mostly leaving django alone in these PRs, but this was needed to fix the issue with sending emails and gets us onto a LTS release.